### PR TITLE
#11698:  call the update script after migrate

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

**Comment by petrus-v** [comment]
I wonder why, travis state still pending but is finished and green. Probably because I've restart the job after pushing [that commit in cluster_lab_test_service on branch qualif](https://github.com/mlfmonde/cluster_lab_test_service/commit/406b5e34cb489571bbf2df3933f7173a8153c876) so reviews are wellcome

**Comment by pyseed** [comment]
> I wonder why, travis state still pending but is finished and green. Probably because I've restart the job after pushing [that commit in cluster_lab_test_service on branch qualif](https://github.com/mlfmonde/cluster_lab_test_service/commit/406b5e34cb489571bbf2df3933f7173a8153c876) so reviews are wellcome

for this commit another small tip input: for L36 echo, ' embedded in '' are not displayed

$ sh -c 'echo "run update data after migrate repo: 'REPO' branch: 'BRANCH'"'
run update data after migrate repo: REPO branch: BRANCH
$ sh -c 'echo "run update data after migrate repo: "REPO" branch: "BRANCH""'
run update data after migrate repo: "REPO" branch: "BRANCH"

you should escape '$REPO' with "$REPO", same thing for BRANCH

**Comment by petrus-v** [comment]
@pyseed hope the answer given on that commit: https://github.com/mlfmonde/cluster_lab_test_service/commit/406b5e34cb489571bbf2df3933f7173a8153c876#commitcomment-31932389 is what you are talking about ?

File: consul/README.rst [hunk 1]
```diff
@@ -39,14 +39,14 @@ You can also use the handler manually to try:
...
+    $ handler.py deploy '{"target": "node1", "branch": "master", "repo": "https://gitlab.example.com/hosting/foobar", "update": true}'
 
 destroying an app::
 
     $ handler.py destroy '{"branch": "master", "repo": "https://gitlab.example.com/hosting/foobar"}'
 
 migrating volumes between two apps::
 
-    $ handler.py migrate '{"target": {"branch": "preprod", "repo": "https://gitlab.example.com/hosting/foobar"}, "branch": "master", "repo": "https://gitlab.example.com/hosting/foobar"}'
+    $ handler.py migrate '{"target": {"branch": "preprod", "repo": "https://gitlab.example.com/hosting/foobar"}, "branch": "master", "repo": "https://gitlab.example.com/hosting/foobar", "update": true}'
```
[the remark this answers is not in the document]
Comment (petrus-v) [reply consul/README.rst]: good point !

**Comment by petrus-v** [comment]
thanks for review ! merging myself

**Pull request merged**